docs/embedded/concepts/app-concepts/containertypes.md

```diff
@@ -22,17 +22,34 @@ The permission mapping between a ContainerTypeID-AppID pair determines the opera
 > [!TIP]
 > To learn more about application architecture, see [Application Architecture](./app-architecture.md).
 
-## SharePoint Embedded Trial Container Types
+
+## TODO: Introduce two types
```

Comment:
This doesn't look complete. Please complete this section before submitting the PR.